### PR TITLE
[9.0][mig][stock_account_quant_merge]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Migrated module from 8.0

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@mreficent @gmeficent can you review?

File: stock_account_quant_merge/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016-17 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
```
Comment (rousseldenis) [inline stock_account_quant_merge/models/stock.py]: Could you set to api, models?

File: stock_account_quant_merge/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/153/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (rousseldenis) [inline stock_account_quant_merge/README.rst]: Please change this

**Review by rousseldenis — CHANGES_REQUESTED** [review]
:+1: Minor comment

**Review by rousseldenis — APPROVED** [review]

**Review by gmeficent — APPROVED** [review]
Functional review :+1:

**Comment by JordiBForgeFlow** [comment]
@andhit-r simple port to v9 of the one you approved in v8

**Pull request merged**